bunq/sdk/json/installation_adapter.py

```diff
@@ -0,0 +1,67 @@
+from bunq.sdk.json import converter
+from bunq.sdk.model.core.id import Id
+from bunq.sdk.model.core.public_key_server import PublicKeyServer
+from bunq.sdk.model.core.session_token import SessionToken
+
+
+class InstallationAdapter(converter.JsonAdapter):
+    # Id constants
+    _ATTRIBUTE_ID = '_id_'
+    _INDEX_ID = 0
+    _FIELD_ID = 'Id'
+
+    # Token constants
+    _ATTRIBUTE_TOKEN = '_token'
+    _INDEX_TOKEN = 1
+    _FIELD_TOKEN = 'Token'
+
+    # Server Public Key constants
+    _ATTRIBUTE_SERVER_PUBLIC_KEY = '_server_public_key'
+    _INDEX_SERVER_PUBLIC_KEY = 2
+    _FIELD_SERVER_PUBLIC_KEY = 'ServerPublicKey'
+
+    @classmethod
+    def deserialize(cls, target_class, array):
+        """
+        :type target_class: Installation|type
+        :type array: list
+
+        :rtype: Installation
+        """
+
+        installation = target_class.__new__(target_class)
+        server_public_key_wrapped = array[cls._INDEX_SERVER_PUBLIC_KEY]
+        installation.__dict__ = {
```

Comment:
maybe I don't understand: can't you just use the normal `__init__` for the `target_class`?

bit ugly to create an empty object with `__new__` first and then setting the dict

Reply:
I'm not 100% sure what you mean here. Is the `installation` variable not created with `__new__` on line 32?

Reply:
yeah but it seems unnecessarily complicated, why not just something like
```python
installation = target_class(foo, bar, baz)
```
instead of creating an empty object through `__new__` and then updating the object's dict.
but perhaps I'm missing something.